### ISSUE TITLE
Nonexistent signal: frame_pre_draw

### ISSUE THREAD

**Issue by qarmin** [godotengine/godot#33135]
**Godot version:**
Godot 3.2 alpha 3 
**OS/device including version:**
Ubuntu 19.10
**Issue description:**
When I run project with Softbody node I have this error:
```
ERROR: _disconnect: Nonexistent signal: frame_pre_draw.
   At: core/object.cpp:1518.
```
**Minimal reproduction project:**
[ZRW.tar.gz](https://github.com/godotengine/godot/files/3778573/ZRW.tar.gz)

**Comment by akien-mga** [comment]
The problem is here, if the mesh is not valid the signal is disconnected, even if it was never connected in the first place:
https://github.com/godotengine/godot/blob/aaa9751de3ef6b39217113aa26f018b25d5878e8/scene/3d/soft_body.cpp#L455-L465

There might be similar issues around other uses of this signal:
```
$ rg frame_pre_draw
servers/visual_server.cpp
2299:   ADD_SIGNAL(MethodInfo("frame_pre_draw"));

servers/visual/visual_server_raster.cpp
100:    VS::get_singleton()->emit_signal("frame_pre_draw");

doc/classes/VisualServer.xml
3935:           <signal name="frame_pre_draw">

scene/2d/cpu_particles_2d.cpp
1035:           VS::get_singleton()->connect("frame_pre_draw", this, "_update_render_thread");
1040:           VS::get_singleton()->disconnect("frame_pre_draw", this, "_update_render_thread");

scene/3d/cpu_particles.cpp
1123:           VS::get_singleton()->connect("frame_pre_draw", this, "_update_render_thread");
1127:           VS::get_singleton()->disconnect("frame_pre_draw", this, "_update_render_thread");

scene/3d/soft_body.cpp
459:            VS::get_singleton()->connect("frame_pre_draw", this, "_draw_soft_mesh");
463:            VS::get_singleton()->disconnect("frame_pre_draw", this, "_draw_soft_mesh");

scene/resources/texture.cpp
2173:   VisualServer::get_singleton()->connect("frame_pre_draw", this, "_update_proxy");
```

**Comment by mikkac** [comment]
Hi I proposed some fix- it's my first contribution, so any tips are welcome :)

**Comment by akien-mga** [comment]
Fixed by #33147.
